vkcs/internal/util/types.go

```diff
@@ -1,4 +1,4 @@
-package vkcs
+package util
 
```

Comment:
should be splitted between vpnaas and networking

Reply:
Done.